README.md

```diff
@@ -40,12 +40,12 @@ This is because there is (reasonable) syntax highlighting for the code examples
 
 There is about as much help as there is nix code in this entire project.
 
-    An important note: if you add a file,
-    nix will not package it unless you add it 
-    to your git staging before you build it...
-    So nvim wont be able to find it...
-    So, run git add before you build,
-    especially when using the wrapRc option.
+> An important note: if you add a file,
```

Comment:
i think this rendering is a bit easier to read, and uses the markdown semantics slightly more appropriately (this isn't a codeblock, it's a note/callout.